ports/geckolib/glue.rs

```diff
@@ -607,7 +606,7 @@ pub extern "C" fn Servo_DeclarationBlock_GetCssText(declarations: RawServoDeclar
 #[no_mangle]
 pub extern "C" fn Servo_DeclarationBlock_SerializeOneValue(
     declarations: RawServoDeclarationBlockBorrowed,
-    property: *mut nsIAtom, is_custom: bool,
+    property: *const nsIAtom, is_custom: bool,
```

Comment:
Could we leave this change and the get_property_name_from_atom change out of this patch?  We'll have to undo them once https://bugzilla.mozilla.org/show_bug.cgi?id=1322098 lands anyway.